modules/appeals_api/spec/support/swagger_shared_components.rb

```diff
@@ -37,6 +37,14 @@ def self.header_params
         example: '706547821',
         schema: { '$ref' => '#/components/schemas/X-VA-SSN' }
       },
+      veteran_icn_header: {
+        in: :header,
+        name: 'X-VA-ICN',
+        required: false,
```

Comment:
A reminder that we'll want to swap this to `true` when we release the Segmented APIs, but we'll need to do so in such a way that it doesn't affect decision_reviews' endpoint docs (even if that's manually by not committing the change to their swagger*.json files).

Reply:
I think we can use `DocHelpers.decision_reviews?` to make the false/true decision programmatically